.bazelrc

```diff
@@ -1,4 +1,4 @@
-build --workspace_status_command=$(pwd)/bazel_stamp_info.sh
+build --workspace_status_command "${PWD}/workspace_status.sh"
```

Comment:
did you test this? I got mine from Brentley himself :)

Reply:
It works. I pulled it from silo